src/components/app/App.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```

Comment:
Let's put in some organizational comments moving forward! Just for file imports, component, any lifecycle methods, render, and any prop checking when we get there.